### DISCUSSION TITLE
how to stop auto complete Permalink on edit

### DISCUSSION THREAD

**Discussion by moathOSA** [area17/twill#1568] [Q&A] [answered]
how can i stop the auto complete for the Permalink on post title edit ?

i'm trying to edit the post title without reflect the change on the permalink, how can i stop the auto complete only on edit mode ?

**Comment by pboivin** [comment, marked as answer]
Hi @moathOSA, have a look at this thread : https://github.com/area17/twill/discussions/983#discussioncomment-888652
